### ISSUE TITLE
Find error case where old manifest has missing entry

### ISSUE THREAD

**Issue by zimeon** [zimeon/ocfl-py#55]
See https://github.com/OCFL/fixtures/pull/84 in which the `v2` inventory `manifest` does not include the fact that the duplicate `v1/content/file-3.txt` is present with digest `b3b2...`

**Comment by zimeon** [comment]
oops:
```
> ./ocfl-validate.py fixtures/1.0/bad-objects/E023_old_manifest_missing_entries
INFO:ocfl.object:OCFL object at fixtures/1.0/bad-objects/E023_old_manifest_missing_entries is VALID
```

**Comment by zimeon** [comment]
Fixed in 1.2.2
